src/main/java/org/zalando/nakadi/repository/zookeeper/ZookeeperConfig.java

```diff
@@ -10,11 +10,11 @@
 public class ZookeeperConfig {
 
     @Bean
-    public ZooKeeperHolder zooKeeperHolder(final Environment environment) {
+    public ZooKeeperHolder zooKeeperHolder(final Environment environment) throws Exception {
```

Comment:
It was in ZooKeeperHolder post construct method before, since I want to use ZooKeeperHolder outside spring context I moved post construct in constructor.